File: APIMASH_CNorris_StarterKit/APIMASH_CNorrisLib/APIMASH_ChuckNorris.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;

/*
* LICENSE: https://raw.github.com/apimash/StarterKits/master/LicenseTerms-SampleApps%20.txt
*/

namespace APIMASH_CNorrisLib
{
    public sealed class APIMASH_CNorris
    {
        private static APIMASH_CNorris _APIMASH_APIName = new APIMASH_CNorris();

        private ObservableCollection<APIMASH_OM_Bindable> _all = new ObservableCollection<APIMASH_OM_Bindable>();
        public ObservableCollection<APIMASH_OM_Bindable> All
        {
            get { return this._all; }
        }

        public static IEnumerable<APIMASH_OM_Bindable> AllItems()
        {
            return _APIMASH_APIName._all;
        }

        public static void Copy(CNorrisJoke response)
        {
            try
            {
                _APIMASH_APIName._all.Clear();

                // 
                // implement copy from OM to BINDABLE OM here
                //
            }
            catch (Exception e)
            {
                throw (e);
            }
        }
    }
}

File: Windows Starter Kits/APIMASH_CNorris_VB_StarterKit/APIMASH_CNorrisVBLib/APIMASH_ChuckNorris.vb
Imports System
Imports System.Collections.Generic
Imports System.Collections.ObjectModel
'
'* LICENSE: https://raw.github.com/apimash/StarterKits/master/LicenseTerms-SampleApps%20.txt
'
Namespace APIMASH_CNorrisLib
    Public NotInheritable Class APIMASH_CNorris
        Private Shared _APIMASH_APIName As New APIMASH_CNorris()

        Private _all As New ObservableCollection(Of APIMASH_OM_Bindable)()

        Public ReadOnly Property All() As ObservableCollection(Of APIMASH_OM_Bindable)
            Get
                Return Me._all
            End Get
        End Property

        Public Shared Function AllItems() As IEnumerable(Of APIMASH_OM_Bindable)
            Return _APIMASH_APIName._all
        End Function

        Public Shared Sub Copy(response As CNorrisJoke)
            Try

                ' 
                ' implement copy from OM to BINDABLE OM here
                '
                _APIMASH_APIName._all.Clear()
            Catch e As Exception
                Throw (e)
            End Try
        End Sub
    End Class
End Namespace
